lib/cuber/cuberfile_validator.rb

```diff
@@ -107,7 +107,7 @@ def validate_env
     
     def validate_health
       return unless @options[:health]
-      @errors << 'health checks must be an http url' unless URI.parse(@options[:health]).kind_of? URI::HTTP
+      @errors << 'health checks must be an http url' unless URI.parse(@options[:health]).kind_of? URI::Generic
```

Comment:
You should have a valid URL in any case, maybe it's a local address like localhost, but you definitely have one. A path is always resolved to a URL.

Reply:
OK, so are you using localhost address? I'm interpreting this as dependent on the pod address, that's why I thought localhost wasn't correct. Keeping a generic path works for me as it will be used on whatever pod IP is needed. Am I reading this wrong?

Reply:
Good question. Cuber in any case uses only the path to make the request, even if you provide a URL. [See the code here](https://github.com/cuber-cloud/cuber-gem/blob/65ab15008188cc8a85edbdcc0f2fa64aab8780cf/lib/cuber/templates/deployment.yml.erb#L196C23-L196C64). The first part of the URL (e.g. domain name) is used only to properly set the `Host` header in the HTTP request for the health check, in case your app needs that.